src/Workspaces/Core/Portable/Formatting/Formatter.cs

```diff
@@ -158,7 +158,7 @@ internal static async Task<Document> FormatAsync(Document document, SyntaxAnnota
         /// <param name="cancellationToken">An optional cancellation token.</param>
         /// <returns>The formatted tree's root node.</returns>
         public static SyntaxNode Format(SyntaxNode node, SyntaxAnnotation annotation, Workspace workspace, OptionSet options = null, CancellationToken cancellationToken = default)
-            => FormatAsync(node, annotation, workspace, options, cancellationToken).WaitAndGetResult(cancellationToken);
+            => FormatAsync(node, annotation, workspace, options, cancellationToken).WaitAndGetResult_CanCallOnBackground(cancellationToken);
```

Comment:
Do we still have offenders after the other ones you fixed?

Reply:
➡️ Based on the number of builds that failed while I worked on this PR, I'm fairly certain we do. Redeeming factor is they only assert in debug builds, and this will eventually get flagged as a diagnostic once https://github.com/Microsoft/vs-threading/issues/344 is implemented.